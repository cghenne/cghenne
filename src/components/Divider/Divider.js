// @flow
import React from 'react';
import './Divider.css';

const Divider = () => {
  return <div className="divider" />;
};

Divider.displayName = 'Divider';

export default Divider;
