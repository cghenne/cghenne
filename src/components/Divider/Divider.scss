@import 'styles/utils.scss';

.divider {
  grid-area: divider;

  background-image: url('data:image/svg+xml;utf8,<svg height="9" width="9" version="1.1" xmlns="http://www.w3.org/2000/svg"><circle cx="4" cy="4" r="2" fill="' + $color-secondary + '" /></svg>');
  background-repeat: repeat-y;
  background-position: center;
}
