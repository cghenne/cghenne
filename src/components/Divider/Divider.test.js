import React from 'react';
import { shallow } from 'enzyme';
import Divider from './Divider';

describe('Divider', () => {
  const wrapper = shallow(<Divider />);

  it('should render without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render the divider class', () => {
    expect(wrapper.find('.divider').exists()).toEqual(true);
  });
});
