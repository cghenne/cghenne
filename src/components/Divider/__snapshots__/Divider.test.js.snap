// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render without crashing 1`] = `
<div
  className="divider"
/>
`;
